src/content/peerconnection/pc1/js/main.js

```diff
@@ -111,9 +113,13 @@ function call() {
   };
   pc2.onaddstream = gotRemoteStream;
 
-  pc1.addStream(localStream);
+  // Created a filtered audio stream.
+  var filteredStream = webAudio.applyFilter(localStream);
+  pc1.addStream(filteredStream);
   trace('Added local stream to pc1');
-
+  // Connect the video stream directly, since the filteredStream contains
+  // only audio. (Is this the right thing to do?)
```

Comment:
I believe it is since you cannot send an audioTrack to webAudio (you must provide the whole mediaStream), if you could you could done the opposite (feed the audio track to WebAudio and then add it back to the original stream).